src/v/pandaproxy/rest/handlers.h

```diff
@@ -11,38 +11,39 @@
 
 #pragma once
 
+#include "pandaproxy/rest/proxy.h"
 #include "pandaproxy/server.h"
 #include "seastarx.h"
 
 #include <seastar/core/future.hh>
 
 namespace pandaproxy::rest {
 
-ss::future<server::reply_t>
```

Comment:
this file could use an alias, to reduce code churn? 

`using server = ctx_server<proxy>`

Reply:
It could, but then including `rest/proxy.h` and `schema_registry/server.h` (i.e., `application.cc`) would conflict.

Reply:
got it. LGTM. looks great. good clean up.